common/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/PayloadConverterImpl.java

```diff
@@ -68,7 +66,18 @@ public T unmarshallFrom(byte[] message) {
             StreamSource source = new StreamSource(is);
             return unmarshaller.unmarshal(source, clazz).getValue();
         } catch (JAXBException e) {
-            throw new MeldingsUtvekslingRuntimeException("Unable to create unmarshaller for " + clazz, e);
+            throw new PayloadConverterException("Unable to create unmarshaller for " + clazz, e);
         }
     }
+
+    public Object marshallToPayload(T message) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)